codalab/lib/worksheet_util.py

```diff
@@ -186,7 +186,7 @@ def get_worksheet_lines(worksheet_info):
     return lines
 
 
-def get_formatted_metadata(cls, metadata, raw=False):
+def get_formatted_metadata(cls, metadata, raw=False, show_hidden=False):
```

Comment:
Is show_hidden used at all?

Reply:
Yeah, if user is root then show_hidden will be true else false.